classes/person.rb

```diff
@@ -2,7 +2,7 @@
 
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] the rental instance variable should have a write attribute as well in order for this relationship to work.